modules/nw-aws-load-balancer-operator-considerations.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+// * networking/aws_load_balancer_operator/understanding-aws-load-balancer-operator.adoc
+
+:_content-type: REFERENCE
+[id="nw-aws-load-balancer-operator-considerations_{context}"]
+= AWS Load Balancer Operator considerations
+
+Review the following limitations before installing and using the AWS Load Balancer Operator (ALBO).
```

Comment:
The naming `ALBO` is quite commonly used inside Red Hat engineering community but I'm not sure whether it's an appropriate acronym for the OCP docs taking into account that the operator seems to be always called by its full name.